src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManagerSpec.groovy

```diff
@@ -103,11 +103,11 @@ class PBSJobManagerSpec extends Specification {
         def parms = JobManagerOptions.create().build()
         TestExecutionService testExecutionService = new TestExecutionService("test", "test")
         PBSJobManager jm = new PBSJobManager(testExecutionService, parms)
-        Method method = PBSJobManager.class.getDeclaredMethod("processQstatOutput", List)
+        Method method = PBSJobManager.class.getDeclaredMethod("processQstatOutput", String)
```

Comment:
Dto. 
If you keep reflection, please pull 
Method method = PBSJobManager.class.getDeclaredMethod("processQstatOutput", String)
method.setAccessible 
method.invoke

to a function. It's code duplication.

Reply:
Please also think, if you can reduce code duplication in the tests. You could maybe store the job manager in class scope?

Reply:
I removed the code duplication